docs/08-pages/01-overview/index.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: Page fundamentals
-description: Explore how pages serve as application foundation with unique URLs, metadata configuration, parameter handling and component containment.
+description: Explore how pages work in Nordcraft, providing unique URLs, metadata configuration, parameter handling and component containment.
 ---
 
-# Page fundamentals
+# Pages in Nordcraft
 
-Pages are the foundation of applications in Nordcraft. They represent distinct screens with unique URLs that users navigate to, creating the overall structure of your web application.
+Pages are the foundation of applications in Nordcraft, representing distinct screens with unique URLs that users navigate to.
 
-# What are pages in Nordcraft
```

Comment:
Are we sure there are no references to this title?

Reply:
I found no references
![image](https://github.com/user-attachments/assets/9c6dfaf0-914d-4297-9ff7-9e37f35f3822)